src/babel-register/register.js

```diff
@@ -9,6 +9,7 @@ if (!process.env.BABEL_CACHE_PATH) {
 // paths that babel-register should ignore
 const ignore = [
   /[\\\/](node_modules|bower_components)[\\\/]/,
+  /[\\\/](kbn_build\\\/dist)[\\\/]/
```

Comment:
the `[\\\/]` component is for either `\` or `/`, which I think you should use between kbn_build and dist